generate_grpc.sh

```diff
@@ -14,7 +14,7 @@ if [ ! -e ${DIR} ]; then
     rm -rf ${DIR}
     mkdir -p $DIR
     wget https://raw.githubusercontent.com/longhorn/longhorn-engine/master/proto/ptypes/common.proto -P $DIR
-    wget https://raw.githubusercontent.com/longhorn/longhorn-engine/master/proto/ptypes/controller.proto -P $DIR
+    wget https://raw.githubusercontent.com/longhorn/longhorn-engine/trim-dev/proto/ptypes/controller.proto -P $DIR
```

Comment:
This is not found `https://raw.githubusercontent.com/longhorn/longhorn-engine/trim-dev/proto/ptypes/controller.proto`, do we need to change this? I think it should be kept at master

Reply:
My fault... Should be `master`.